Docs/developer_guide/parameter_nodes.md

```diff
@@ -331,6 +337,8 @@ The following methods are available for `ObservedList`:
 Similarly for a parameter of `list[list[type]]`, an `ObservedList[ObservedList[type]]` is returned.
 When calling `param.values.append`, `param.values[index] = object` or `+=` in these cases, a normal `list` can be passed in and it will be converted to an `ObservedList`.
 
+There are similar mechanisms in place for `tuple`, `dict`, and the `parameterPack`s.
```

Comment:
`parameterPack` is missing from this PR.

Reply:
Good catch. Will fix docs.

Reply:
Fixed.